setup.py

```diff
@@ -21,6 +21,7 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```

Comment:
No, I would just be more specific with the classifiers, i.e.

```
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
 ```

We could also test with more versions on Travis. Reportlab supports 2.7 and 3.4 officially.